Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -294,6 +294,24 @@ return
 					},
 					Notes = "Returns whether the specified block type is solid.",
 				},
+				IsSkylightDispersant =
+				{
+					IsStatic = true,
+					Params =
+					{
+						{
+							Name = "BlockType",
+							Type = "number",
+						},
+					},
+					Returns =
+					{
+						{
+							Type = "boolean",
+						},
+					},
+					Notes = "Returns whether skylight may pass through a block of the specified type unimpeded.",
```

Comment:
Isn't this the other way around? If this returns `true`, it means that the skylight will NOT pass through this block.

Reply:
Updated.